types/difflib/index.d.ts

```diff
@@ -1,12 +1,27 @@
 // Type definitions for difflib 0.2
 // Project: https://github.com/qiao/difflib.js
-// Definitions by: majames <https://github.com/majames>, Hugues Le Gendre <https://github.com/hlegendre>
+// Definitions by: majames <https://github.com/majames>
+//                 Hugues Le Gendre <https://github.com/hlegendre>
+//                 Antonio Min <https://github.com/antoniomin>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 3.4
 
 export class SequenceMatcher<T> {
     constructor(isjunk: (() => boolean) | null, left: T, right: T, autojunk?: boolean);
+    setSeqs(left: T, right: T): (char: string) => boolean | undefined;
+    setSeq1(left: T): null | undefined;
+    setSeq2(right: T): (char: string) => boolean | undefined;
```

Comment:
I think it is `((char: string) => boolean) | undefined` instead of `(char: string) => boolean | undefined`, don't you think?
[This line](https://github.com/qiao/difflib.js/blob/e11553ba3e303e2db206d04c95f8e51c5692ca28/lib/difflib.js#L284) returns an undefined, whereas the type provided implies a function that return either a boolean or undefined...

Reply:
@hlegendre Thanks for review! i changed it